default.nix

```diff
@@ -80,5 +80,5 @@ in rec {
         inherit (pkgsVersions.latest) solidityPackage;
```

Comment:
I guess you can't. When is it useful?

If often, we can special case that too.

Basically I'm trying to avoid invocations like `nix-shell --arg pkgs 'with import (fetchTarball https://github.com/makerdao/nixpkgs-pin/tarball/master) { dappSrcOverrides = { current = fetchTarball https://github.com/dapphub/dapptools/tarball/master; }; }; pkgs'` ;)